reference-stores/nuxt/queries/collection.js

```diff
@@ -2,9 +2,17 @@ import { PRODUCT_QUERY_FRAGMENT } from './product';
 
 export const COLLECTION_PRODUCTS_QUERY = `
   query CollectionProducts($handle: String!, $after: String!){
-    collections: productCollections(filter: { handles: [$handle] }){
-      products(first: 12, after: $after){
-        ${PRODUCT_QUERY_FRAGMENT}
+    collections: allProductCollections(filter: { handles: [$handle] }){
+      edges {
+				node {
+          products: productConnection(first: 12, after: $after){
+            pageInfo {
+              endCursor
+              hasNextPage
+            }      
+            ${PRODUCT_QUERY_FRAGMENT}
+          }
+        }
       }
```

Comment:
Looks like there's some funky whitespace

```suggestion
      edges {
        node {
          products: productConnection(first: 12, after: $after){
            pageInfo {
              endCursor
              hasNextPage
            }
            ${PRODUCT_QUERY_FRAGMENT}
          }
        }
      }
```